sdk/trace/span.go

```diff
@@ -444,24 +421,17 @@ func truncate(limit int, s string) string {
 // If this method is called while panicking an error event is added to the
 // Span before ending it and the panic is continued.
 func (s *recordingSpan) End(options ...trace.SpanEndOption) {
-	// Do not start by checking if the span is being recorded which requires
-	// acquiring a lock. Make a minimal check that the span is not nil.
 	if s == nil {
 		return
 	}
 
-	// Store the end time as soon as possible to avoid artificially increasing
-	// the span's duration in case some operation below takes a while.
-	et := monotonicEndTime(s.startTime)
```

Comment:
I think this is equivalent to time.Now since `time.Now` returns a monotonic time.